internal/lcow/process.go

```diff
@@ -57,7 +54,7 @@ type ProcessOptions struct {
 //
 // It is the responsibility of the caller to call Close() on the process returned.
 
-func CreateProcess(opts *ProcessOptions) (*hcs.Process, *ByteCounts, error) {
+func CreateProcess(opts *ProcessOptions) (cow.Process, *ByteCounts, error) {
```

Comment:
This is only called by a test method. Can we remove it?

Reply:
That's not true. It's called by scratch.go.

Reply:
Oh its used inside the package. This makes me sad its basically pure duplication. Oh well